src/Document/parseBZWDocument.ts

```diff
@@ -40,6 +45,13 @@ const ObjectBuilders: Record<string, ObjectBuilder> = {
     parsers: MeshFaceProperties,
     finalize: noop,
   },
+  link: {
+    factory: newITeleporterLink,
+    parsers: TeleporterLinkProperties,
+    finalize: (link: ITeleporterLink) => {
+      link.name = link._infoString;
+    },
```

Comment:
Also, according to the [bzw Man page](https://dspinellis.github.io/manview/?src=https://raw.githubusercontent.com/BZFlag-Dev/bzflag/3be72c3aeba3f5f5e118a8279fb885333dc2b5b7/man/bzw.5.in), it appears that `link` uses the `name` attribute; so this could go back to be a `noop` call.

```
The Link object
Adds a route to teleport a tank between two teleporters.
# Teleporter names are terminated with either :f (forward)
# or :b (backwards). The forwards link points to 0 degrees,
# and the backwards link points to 180. Links are made by
# pattern matching the teleporter names. The '*' and '?'
# globbing characters can be used to make multiple matches.
# If there are multiple matches for the "to" link, then the
# destination will be selected randomly between the matches.
# in −game.

# NOTE: bzfs −d −d −d −d will print the linkage table.
link
name example_link
# this will link all teleporters randomly to all other teleporters
from *
to *
end

# or, to link between known teleporters examp_tele1(front) and examp_tele2(back)
link
name example_realLink
from examp_tele1:f
to examp_tele2:b
end
```

Reply:
Good catch. I had that set up correctly, but the unit test failed because I wrote the bzwBody wrong. So I thought I needed that.